app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
```

Comment:
- [ ] You only have the user as the creator right now, there should also exist as Association for the user as an Atendee